src/toolkit/facility_cost.h

```diff
@@ -0,0 +1,45 @@
+#ifndef CYCLUS_SRC_TOOLKIT_FACILITY_COST_H_
+#define CYCLUS_SRC_TOOLKIT_FACILITY_COST_H_
+
+
+#include <string>
+#include "cyclus.h"
+
+namespace cyclus {
+namespace toolkit {
+
+/// @class FacilityCost
+/// The FacilityCost class is a class that handles basic cost data for
+/// Facilities within cyclus. The inner workings of this class are TBD,
+/// but it will first serve as a very basic showcase of code injection into
+/// Facility header files for state variables which can be common to all
+/// Facilities.
+
+class FacilityCost {
+ public:
+
+  /// The default constructor for FacilityCost. This creates an object
+  /// with all costs set to 0.0.
+  FacilityCost() : capital_cost_(0) {}
```

Comment:
Here's what I was thinking for combining these
```suggestion
  FacilityCost(double capital_cost = 0) : capital_cost_(capital_cost) {}
```

Reply:
Oh! I was being dense. This makes way more sense than what I thought you meant.